### ISSUE TITLE
Tailwindcss not working with esbuild (scss)

### ISSUE THREAD

**Issue by spock123** [angular/angular-cli#25362]
### Which @angular/* package(s) are the source of the bug?

core, Don't known / other

### Is this a regression?

Yes

### Description

Creating a new project (standalone) with V16, and installing TailwindCSS, does not seem to work when switching to EsBuild.

### Please provide a link to a minimal reproduction of the bug

https://github.com/spock123/angular-tailwind-esbuild.git

### Please provide the exception or error you saw

Clone repository, which is a basic new angular application, created with _pnpm_ package manager.
_TailwindCSS_ is then installed and configured.

```
pnpm install
npm run start
```

Tailwind classes are applied, both from inline classes and using @apply in the .scss file.

Now change to ESBuild in angular.json

Tailwind classes are no longer applied from the scss file.

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI: 16.0.3
Node: 18.15.0
Package Manager: pnpm 8.1.0
OS: darwin arm64

Angular: 16.0.0
... animations, common, compiler, compiler-cli, core, forms
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1600.3 (cli-only)
@angular-devkit/build-angular   16.0.3
@angular-devkit/core            16.0.3 (cli-only)
@angular-devkit/schematics      16.0.3 (cli-only)
@angular/cli                    16.0.3
@schematics/angular             16.0.3 (cli-only)
rxjs                            7.8.0
typescript                      5.0.2
```

### Anything else?

Updating to ng16.0.5 did not help.

**Comment by JoostK** [comment]
This is a duplicate of #25169 and has been implemented in #25279, which will be available in 16.1.

**Comment by spock123** [comment]
@JoostK I*m sorry I did not see this. Thank you so much for the update. Cheers
